src/pages/Home.tsx

```diff
@@ -28,19 +37,23 @@ const Home: () => JSX.Element = () => {
             marginX={4}
             marginBottom={2}
             marginTop={['4', null, '0', null]}
-            height={['150px', null, '100%', null]}
+            height={['100%', null, '100%', null]}
             borderRadius={6}
           >
-            <Heading>Current Sensor Box</Heading>
-            <Text>AQI: {currentSensor}</Text>
+            {displayDial && <AQIDial currentReading={currentSensor} />}
```

Comment:
Update this code to be something like:
`{currentReading ? 
<AQIDial currentReading={currentSensor} />
:
<Text marginTop={[null, null, '20%', null]}>
  Click a sensor on the map to see its AQI value here!
</Text>
}`

Reply:
This would replace lines 43 to 48